index.html

```diff
@@ -380,4 +380,4 @@ <h2>Signature Info</h2>
     </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
Sorry my editor added a newline at the end of this file. It's technically correct to end the file with a newline but I can remove if you really want min-diff.